draft-ietf-gnap-core-protocol.md

```diff
@@ -204,29 +194,37 @@ role of the AS as defined by the protocol.
 In addition to the roles above, the protocol also involves several 
 elements that are acted upon by the roles throughout the process.
 
+Attribute
+: characteristics related to an end-user.
+
 Access Token
-: A credential representing a set of access rights
-delegated to the client instance. The access token is created by the AS, consumed
-and verified by the RS, and issued to and carried by the client instance. The contents
-and format of the access token are opaque to the client.
+: digitally signed data that contains specific rights and/or attributes.
+
+    Note: an access token can be first issued to an end-user (usually requiring his authentication) and subsequently refreshed.
+
+Claim
+: statement about a subject.
```

Comment:
I don't think this is a helpful definition, especially if we only use it in the context of "subject information" below. It's a very general term to the point of being almost meaningless.

Reply:
currently, it's indeed used only in subject information. The idea was to say that it is a statement made by someone, not a fact. Can merge into the main definition if we don't find that useful.

Reply:
removed